packages/client/hmi-client/src/components/workflow/tera-stratify.vue

```diff
@@ -248,8 +248,33 @@ const modelConfiguration = ref<ModelConfiguration>();
 const model = ref<Model | null>(null);
 const typedBaseModel = ref<Model | null>(null);
 const typedStrataModel = ref<Model | null>(null);
-const stratifiedModel = ref<Model>();
+const stratifiedModel = ref<Model | null>();
 const numUntypedNodes = ref();
+const initialState = {
+	stratifyView: StratifyView.Input,
+	stratifyStep: 1,
+	strataType: null,
+	labels: null,
+	strataModel: null,
+	model: null,
+	typedBaseModel: null,
+	typedStrataModel: null,
+	stratifiedModel: null,
+	numUntypedNodes: -1
+};
+
+function setState(state) {
+	stratifyView.value = state.stratifyView;
+	stratifyStep.value = state.stratifyStep;
+	strataModel.value = state.strataModel;
+	strataType.value = state.strataType;
+	labels.value = state.labels;
+	model.value = state.model;
+	typedBaseModel.value = state.typedBaseModel;
+	typedStrataModel.value = state.typedStrataModel;
+	stratifiedModel.value = state.stratifiedModel;
+	numUntypedNodes.value = state.numUntypedNodes;
+}
```

Comment:
Probably `restoreState` is a better name here.

Also note generally speaking this is not how "state" is treated in the workflow, they actually have typing information and are initialized in a specific manner. I don't think it is worthwhile to change to a proper state usage now since we are wrapping up the catlab stuff, but please make sure you understand the convention and paradigm we are using.